etcd-client/src/watch/mod.rs

```diff
@@ -189,6 +185,7 @@ impl Watch {
         Self { client, tunnel }
     }
 
+    /// # Panics
```

Comment:
Please add scenarios in which panicking occurs.

Reply:
Done.